coalib/output/ConsoleInteraction.py

```diff
@@ -649,88 +649,106 @@ def show_enumeration(console_printer,
     console_printer.print()
 
 
-def show_bear(console_printer, bear, sections, metadata):
+def show_bear(bear,
+              sections,
```

Comment:
It'd be nice to mention the data structure for sections. I believe it's list here?